### ISSUE TITLE
Document history API

### ISSUE THREAD

**Issue by mnquintana** [AmpersandJS/ampersand-router#48]
Some currently undocumented methods of the history API (like `getFragment`) are very useful for users of this package - any change you'd consider documenting them publicly? :smile:

**Comment by wraithgar** [comment]
Yes! Absolutely.  Documentation is extremely helpful and important.  The general rule of thumb in the past has been that if it's documented, it should have test coverage.  A quick scan of the existing tests seems to indicate that this is already the case with `getFragment`.

If this is something you're considering, please know that documentation updates are *very* easy to get merged in.  They only require one person on the team to see it, like it, and merge it in.

If you don't want to do this, no problem. We will leave this issue here and try to get some momentum on it.  Thanks for this suggestion.
